package.json

```diff
@@ -57,31 +57,31 @@
         "@commitlint/config-conventional": "^17.3.0",
         "@fremtind/browserslist-config-jkl": "^1.0.3",
         "@fremtind/stylelint-config-jkl": "^6.0.4",
-        "@frsource/cypress-plugin-visual-regression-diff": "^3.1.6",
+        "@frsource/cypress-plugin-visual-regression-diff": "^3.2.3",
```

Comment:
Mulig det har kommet en regresjon her. Forstår ikke hvorfor vi endte opp med regenererte snapshots i denne PRen.

Reply:
Dette er working as intended (https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/issues/197) @piofinn

Reply:
La inn mine to cent som en kommentar. Vi får vurdere hvor ofte vi oppdaterer akkurat denne pakka da, hvis den regenererer og committer noen MB i bilder hver gang.